featurebyte/worker/task/base.py

```diff
@@ -32,58 +33,20 @@ def __init__(  # pylint: disable=too-many-arguments
         task_id: UUID,
         payload: dict[str, Any],
         progress: Any,
-        app_container: LazyAppContainer,
+        user: User,
+        persistent: Persistent,
+        storage: Storage,
+        temp_storage: Storage,
     ):
         if self.payload_class == BaseTaskPayload:
             raise NotImplementedError
         self.task_id = task_id
         self.payload = self.payload_class(**payload)
         self.progress = progress
-        self.app_container = app_container
-
-    @property
```

Comment:
dont need these properties anymore since we can just inject dependencies as per normal like other services/controllers